sklearn/svm/classes.py

```diff
@@ -431,7 +431,9 @@ class SVC(BaseSVC):
 
     The implementation is based on libsvm. The fit time complexity
     is more than quadratic with the number of samples which makes it hard
-    to scale to dataset with more than a couple of 10000 samples.
+    to scale to dataset with more than a couple of 10000 samples. For large
+    datasets consider using :class:`LinearSVC` or :class:`SGDClassifier`
```

Comment:
we need the full module path for the link to work, no? sklearn.linear_model.SGDClassifier